files/justfiles/secureblue.just

```diff
@@ -700,6 +702,11 @@ audit-secureblue:
                 warnings+=("> $f has device=all permission,")
                 warnings+=("> granting access to GPU, input devices, raw USB, and virtualization,")
                 warnings+=("> and introducing a vector for sandbox escapes!")
+                warnings+=("> To remove it use Flatseal or run:")
+                warnings+=("> 'flatpak override -u --nodevice=all $f'")
+                warnings+=("> We also recommend you enable device=dri to grant GPU access")
```

Comment:
(nit) "to grant GPU access, if needed by the application"

Reply:
added, but said "app" instead of "application to keep line width <80